CONTRIBUTING.md

```diff
@@ -1,3 +1,7 @@
+# <a name="contributing">Contributing to meshery</a>
```

Comment:
@subhamkrai given that these *are* the master contribution instructions, I don't know that we need this section here, but rather need this section in the adapter repos (in their contributing.mds).